circuit-conversations-list.js

```diff
@@ -0,0 +1,210 @@
+const template = document.createElement('template');
+const initHtml = '<select class="select-wide" part="conversation-list" id="conversations-list"></select>';
```

Comment:
What about all the default styles, host styles, inherited styles etc?
See the other components. It should look a bit like circuit by default.